### PR TITLE
use keeper when known for slack attr

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner @cvializ 

before: 
screen shot 2016-01-06 at 7 10 14 pm

after:
screen shot 2016-01-06 at 7 09 39 pm

### CODE REVIEW

**Comment by andrewconner** [comment]
:trophy: Nice!

File: kifi-backend/modules/shoebox/angular/src/keep/keepCardAttribution.tpl.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
 <div class="kf-keep-card-attribution">
   <a class="kf-keep-card-attribution-keeper-pic" ng-if="::keep.user && !keep.sourceAttribution.slack" ng-href="{{::keep.user|profileUrl}}"
      ng-attr-style="background-image:url({{::keep.user|pic:100}})"></a>
-  <span class="svg-slack kf-keep-card-attribution-keeper-pic kf-keep-card-attribution-no-border" ng-if="::keep.sourceAttribution.slack"></span>
+  <a class="kf-keep-card-attribution-keeper-pic" ng-if="::keep.sourceAttribution.slack && keep.sourceAttribution.kifi" ng-href="{{::keep.sourceAttribution.kifi|profileUrl}}"
+     ng-attr-style="background-image:url({{::keep.sourceAttribution.kifi|pic:100}})"></a>
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/keep/keepCardAttribution.tpl.html]: this is not quite how `ng-attr-*` works. you want `ng-style="{ 'background-image': 'url(' + (keep.sourceAttribution.kifi | pic:100) + ')' }"`

File: kifi-backend/modules/shoebox/angular/src/keep/keepCardAttribution.tpl.html [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+  <span class="svg-slack kf-keep-card-attribution-keeper-pic kf-keep-card-attribution-no-border" ng-if="::keep.sourceAttribution.slack && !keep.sourceAttribution.kifi"></span>
   <div class="kf-keep-card-attribution-message">
     <!--This is wrapped in a span to make sure there is a space after the name in safari :(-->
     <span ng-if="::keep.user && !keep.sourceAttribution.slack"><a class="kf-link"
        ng-href="{{::keep.user|profileUrl}}">{{::keep.user|name}}
     </a></span>
-    <span ng-if="::keep.sourceAttribution.slack">@<span ng-bind="keep.sourceAttribution.slack.message.username"></span> added this</span>
+    <span ng-if="::keep.sourceAttribution.slack"><a class="kf-link" ng-href="{{keep.sourceAttribution.kifi | profileUrl}}"><span ng-bind="keep.sourceAttribution.kifi ?
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/keep/keepCardAttribution.tpl.html]: you don't need the extra span for this `ng-bind`, you can put it on the `<a>` tag

**Comment by cvializ** [comment]
good stuff :octopus: !

File: kifi-backend/modules/shoebox/angular/src/keep/keepCardAttribution.tpl.html (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/keep/keepCardAttribution.tpl.html]: wow, that's way different. seems like what's there works though? will do

File: kifi-backend/modules/shoebox/angular/src/keep/keepCardAttribution.tpl.html (hunk 2 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/keep/keepCardAttribution.tpl.html]: noice, thanks

File: kifi-backend/modules/shoebox/angular/src/keep/keepCardAttribution.tpl.html (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/keep/keepCardAttribution.tpl.html]: I'm not sure why it works here, but `ng-attr-*` is used when you want an attribute to appear only if the evaluated value is not undefined. or if the attributes have special handling by the DOM, like some SVG attributes

**Pull request merged**